content/docs/1.7.0/advanced-resources/deploy/node-selector.md

```diff
@@ -10,12 +10,18 @@ In this case, you can set the node selector to restrict Longhorn to only run on
 For more information about how node selector work, refer to the [official Kubernetes documentation.](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
 
 # Setting up Node Selector for Longhorn
-Longhorn system contains user deployed components (e.g, Manager, Driver Deployer, UI) and system managed components (e.g, Instance Manager, Engine Image, CSI Driver, etc.)
+Longhorn consists of user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI) and system-managed components (for example, Instance Manager, Backing Image Manager, Share Manager, CSI Driver, and Engine Image).
 You need to set node selector for both types of components. See more details below.
 
 ### Setting up Node Selector During installing Longhorn
-1. Set node selector for user deployed components (Manager, UI, Driver Deployer)
-   * If you install Longhorn by Rancher 2.5.x, you need to click `Edit as YAML` in Rancher UI and copy this values into the YAML:
+1. Set the node selector for user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI).
+   * If you install Longhorn through Rancher 2.5.x, you must copy and paste the following parameters into the YAML on the Rancher UI (click **Edit as YAML** during the installation) to apply the value to all user-deployed components.
```

Comment:
`Rancher 2.5.x` is quite old version. Let's mention Rancher w/o any specific version.